scala/com/hello/Main.scala

```diff
@@ -0,0 +1,7 @@
+package com.hello;
```

Comment:
To keep this parallel to the Java example, let's rename this to `com.engflow.example` and move to that directory. Also add a test.